### PR TITLE
More logging tweaks

### PR DESCRIPTION
**Pull request by dmannarino**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Make sure you are requesting to pull a topic/feature/bugfix branch (right side). Don't request your master!
- [x] Make sure you are making a pull request against the develop branch (left side). Also you should start your branch off our develop.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.

## Pull request type

 

 

Please check the type of change your PR introduces:
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

- Version update jobs no longer report success to Slack (regression)
- Sync jobs silently ignore unknown sync types

Issue Number: GTC-2071

## What is the new behavior?

- Version update jobs DO report success to Slack
- Unknown sync types are reported (as a warning) to Slack
-

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/Dockerfile [hunk 1]
```diff
@@ -11,7 +11,7 @@ COPY . $WORKDIR
 # installing dependencies to build package
 RUN pip install . -t python
 
-# change 31812
+# change 31815
```
Comment (gtempus) [inline src/Dockerfile:14]: I forget. Why is this necessary? I seem to remember it having to do with Terraform. 🤔

**Review by gtempus — APPROVED** [review]
Better to have more info than less when we need the context. 👍 
I just had one quick question...

File: src/Dockerfile (hunk 1 above)
Comment (dmannarino) [reply src/Dockerfile:14]: The reasons are slightly unclear why (to me, who hasn't tried to figure it out in a long while), but basically if there isn't a change in the Dockerfile (or its hash, most likely) then Terraform doesn't think it needs to replace a particular resource (which one, I forget, but it's important) in AWS. It's annoying, but we are lazy. :)
